napari/utils/misc.py

```diff
@@ -69,12 +71,26 @@ def running_as_bundled_app() -> bool:
 def running_as_constructor_app() -> bool:
     """Infer whether we are running as a constructor bundle."""
     return (
-        Path(sys.executable).parent / ".napari_is_constructor_bundled"
-    ).exists() or (
-        Path(sys.prefix).parent / ".napari_is_constructor_bundled"
+        Path(sys.prefix).parent.parent / ".napari_is_bundled_constructor"
     ).exists()
 
 
+def bundle_conda_dependencies() -> List[str]:
```

Comment:
I don't understand why this is done this way rather than e.g. parsing the setup.cfg? Could you add a more elaborate comment about this design in the docstring?

Reply:
@jni, yes I can add an extra description. The idea is that we want to preserve the initial napari environment in the bundle app when installing plugins. 

Since there will not always be a napari conda package to do this, I am grabbing the dependencies from the `conda-meta/napari-x.x.x-zzz.json` file so that whenever a plugin is installed inside the bundled app, we also ensure we pass the same napari dependencies, so that installing a plugin does not change a dependency in an unexpected way.

We will enhance this in the next release cycle of napari, so we can define some base constraints we always need to ensure (numpy, scipy, daks, scikits, python etc)

Reply:
Also @jni this is only for the constructor based bundle. This has no effect for the briefcase bundle or the normal napari, that is why we are suing the conda metadata, not the setup.cfg one.

Reply:
Oooooh, got it. 👍 Yeah if you put that comment in the docstring verbatim I will be much happier merging this. But perhaps I will just merge and then you can add in subsequent PR, if you are not around right now.

Reply:
I did but forgot to push 🤦🏻‍♂️ 

Gimme 5

Reply:
Added @jni!